server/views/partials/additionalLicenceConditionsSummary.njk

```diff
@@ -35,29 +35,24 @@
         }), additionalLicenceConditions) %}
     {% endif %}
 
-    {% for condition in additionalConditions %}
-        {% if condition.uploadSummary.length %}
-            {% set actionLink =  "/licence/create/id/" + licence.id + "/additional-licence-conditions/condition/" + condition.code + "/file-uploads" %}
```

Comment:
Is removing this intentional? Will the new code not mean that clicking the 'change' link won't appear for MEZ with multiple maps? (and if there's only one upload, the change link will no longer go to `/file-uploads`?)

Reply:
this list of additionalConditions will never include any conditions that have upload information, those are dealt with the `conditionsWithUploads` loop below